gcp-pubsub/src/main/java/io/micronaut/gcp/pubsub/bind/PubSubBodyBinder.java

```diff
@@ -37,9 +41,34 @@
 @Singleton
 public class PubSubBodyBinder implements PubSubAnnotatedArgumentBinder<MessageBody> {
 
+    private final ConversionService conversionService;
+
     private final PubSubMessageSerDesRegistry serDesRegistry;
 
+    /**
+     * Constructs a PubSub body binder instance.
+     *
+     * @deprecated
```

Comment:
fill `@deprecated` javadoc

Reply:
That's what the next line was intended for (and it does render than way). I removed the line break to make it more obvious, and also added a further link to the new constructor.

Reply:
I think you should write 

`@deprecated Use {@link PubSubBodyBind(ConversionService, PubSubMessageSerDesRegistry)} instead.